pages/querying/expressions.mdx

```diff
@@ -34,105 +35,145 @@ When automating the queries for Memgraph, it comes in handy to change only some
 parts of the query. Usually, these parts are values that are used for filtering
 results or similar, while the rest of the query remains the same.
 
-Parameters allow reusing the same query but with different parameter values. The
-syntax uses the `$` symbol to designate a parameter name. We don't allow old
-Cypher parameter syntax using curly braces. For example, you can parameterize
-filtering a node property:
+Parameters allow reusing the same query but with different parameter values.
+They are indicated using the `$` symbol followed by the parameter name (e.g.,
+`$name`, `$limit`). 
+
+<Callout type="info"> 
+   Memgraph **does not** support the old Cypher syntax using curly braces (e.g., 
+   `{param}`). 
+</Callout>
+
+Below are some examples of the usage of parameters in Memgraph.
+
+### Common examples
+
+**Filtering node properties**
 
 ```cypher
-MATCH (node1 {property: $propertyValue}) RETURN node1;
+MATCH (node {property: $propertyValue}) RETURN node;
 ```
 
-If you want to update a property of a node, you can use the following:
+**Updating node properties**
 
 ```cypher
 MATCH (n) SET n.propertyValue = $propertyNewValue;
 ```
 
-You can use parameters instead of any literal in the query. For example, if you want to filter nodes based on a specific label, you can execute the following query:
+**Using parameters in LIMIT clauses**
+
+Memgraph also supports parameters in `LIMIT` clauses. For example:
+
+```cypher
+MATCH (n) 
+RETURN n 
+LIMIT $limit;
+```
+
+**Using parameters with `LOAD CSV`**
+
+Additionally, Memgraph supports parameters for LOAD CSV queries. For instance:
+
+```cypher
+LOAD CSV FROM $file AS row 
+CREATE (:Node {property: row.property});
+```
+
+### Label parameterization
+
+Starting from version 3.1, Memgraph allows node labels to be created using
+expressions of type `String` or `List[String]`. 
+
+**Static label parameterization**
+
+You can use parameters instead of any literal in the query. For example, if you
+want to filter nodes based on a specific label, you can execute the following
+query:
 
 ```cypher
 MATCH (n:$label)
 RETURN n;
 ```
 
-This syntax supports parameters of type `String` and `List[String]`, allowing a
-node to have one or multiple labels assigned.
 
-Using parameters as property maps is partially supported, it isn't supported in `MATCH` nor `MERGE`
-clause. For example, the following query is illegal:
+**Dynamic label parameterization**
+
+You can dynamically assign multiple labels to a node using parameters:
 
 ```cypher
-MATCH (n $propertyMap) RETURN n;
+MATCH (n)
+SET n:$labels;
 ```
 
-but this is supported:
+Given the following parameter:
 
-```cypher
-CREATE (n $propertyMap) RETURN n;
+```json
+{
+  "labels": ["Person", "Student"]
+}
 ```
 
-Memgraph also supports parameters in LIMIT clauses. For example:
+### Using property maps
+
+Memgraph supports property maps **in some clauses**. For example:
+
+**Not supported (e.g., in `MATCH` or `MERGE`):**
```

Comment:
I would first write where it's supported (+ you emphasized it's supported in some clauses). 
Then below, I would write where it's not supported.